src/main/kotlin/no/nav/klage/oppgave/config/OpenApiConfig.kt

```diff
@@ -37,4 +38,16 @@ class OpenApiConfig {
             .tags(Tag("kabal-api", "API for saksbehandlere ved klageinstansen"))
     }
 
+    @Bean
+    fun apiDokumenter(): Docket {
+        return Docket(DocumentationType.OAS_30)
+            .select()
+            .apis(RequestHandlerSelectors.basePackage(DokumentUnderArbeidController::class.java.packageName))
+            .build()
+            .pathMapping("/")
+            .groupName("internal-dokumenter")
```

Comment:
Er kanskje ikke nødvendig med en egen gruppering for dokumenter? Tag grupperer jo også. Men du kan vel deploye og se hva som blir pent :)

Reply:
Jeg har gjort et nytt forsøk!